.github/workflows/validate-schema.yml

```diff
@@ -0,0 +1,23 @@
+name: Validate schema
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  validate-json-schema:
+    name: v${{ matrix.schema }} schema
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        schema: [12, 13, 14-draft]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install validator
+        run: npm install ajv-cli
+      - name: Download JSON meta-schema
+        run: wget -O meta-schema https://json-schema.org/draft-07/schema
```

Comment:
I guess we could preinstall ajv-cli and pre download the meta-schema, but since it runs quite fast we can still do this at a later point.